### ISSUE TITLE
Enhance the Order of Text for Screen Reader

### ISSUE THREAD

**Issue by xulihang** [xulihang/ImageTrans-docs#553]
As Screen Reader only returns the text of an area, we cannot do post-processing like adjusting the order of the lines after the result is given.

So we have to make sure the order is adjusted correctly during OCR.

A "Correct text order" option is added to Screen Reader. It will get the text with locations and sort the text.

In addition, if you need to keep the paragraph info, you can disable "Auto remove line breaks":

![image](https://github.com/xulihang/ImageTrans-docs/assets/5462205/d0249186-6fc4-4679-9a3f-f506f0235779)

You can also enable merging based on punctuations to get meaningful text groups: https://github.com/xulihang/ImageTrans-docs/issues/552

**Comment by xulihang** [comment]
Added in v2.8.0.
